### PR TITLE
New whitelisted ruby formulae

### PR DESCRIPTION
**Pull request by bennettgo**
new methods due to workato/workato#22951

### CODE REVIEW

File: docs/formulas/string-formulas.md [hunk 1]
```diff
@@ -433,6 +456,22 @@ Converts a value of another data type into a string data type.
 
 ---
 
+## ordinalize
+
+Turns a number into an ordinal string used to denote the position in an ordered sequence such as 1st, 2nd, 3rd, 4th.
+
+###Example
```
Comment (graceliching) [inline docs/formulas/string-formulas.md]: missing space here '### Example'

File: docs/formulas/string-formulas.md [hunk 2]
```diff
@@ -272,6 +272,18 @@ This function replaces all occurrence of the first input value, with the second
 
 ---
 
+## sub
+
+Substitute the first occurrence of a pattern with value.
```
Comment (graceliching) [inline docs/formulas/string-formulas.md]: >gsub: This function replaces all occurrence of the first input value, with the second input value, within the string. This function is case-sensitive - make sure to downcase or upcase before comparison if you are concerned about case sensitivity.

We should use similar language to explain sub and gsub -- 

## gsub

Substitute **all** occurrences of a string/pattern with a specified value. This function is case-sensitive.

## sub

Substitute the **first** occurrence of a string/pattern with a specified value. This function is case-sensitive.

File: docs/developing-connectors/sdk/methods.md [hunk 3]
```diff
@@ -134,6 +146,14 @@ Furthermore, you'll be able to declare personal reusable methods to use in any b
       <td>each</td>
       <td>Basic iterator<br><code>[1, 2, 3].each { |i| puts i }</code></td>
     </tr>
+    <tr>
+      <td>each_byte</td>
+      <td>Passes each byte in str to the given block, or returns an enumerator if no block is given. <a href="https://apidock.com/ruby/v2_5_5/String/each_byte">More details here.</a></td>
```
Comment (eeshansim) [inline docs/developing-connectors/sdk/methods.md]: use string instead of str

File: docs/developing-connectors/sdk/methods.md [hunk 4]
```diff
@@ -134,6 +146,14 @@ Furthermore, you'll be able to declare personal reusable methods to use in any b
...
+    </tr>
+    <tr>
+      <td>each_char</td>
```
Comment (eeshansim) [inline docs/developing-connectors/sdk/methods.md]: use string instead of str

File: docs/formulas/string-formulas.md (hunk 2 above)
Comment (bennettgo) [reply docs/formulas/string-formulas.md]: Changed sub's description to look more like gsub

File: docs/formulas/string-formulas.md [hunk 5]
```diff
@@ -274,7 +274,7 @@ This function replaces all occurrence of the first input value, with the second
 
 ## sub
 
-Substitute the first occurrence of a pattern with value.
+This function replaces the first occurrence of the first input value, with the second input value, within the string. This function is case-sensitive - make sure to downcase or upcase before comparison if you are not concerned about case sensitivity.
```
[the remark this answers is not in the document]
Comment (graceliching) [reply docs/formulas/string-formulas.md]: need to change the gsub one too

**Pull request merged**